bin/templates/cordova/lib/prepare.js

```diff
@@ -26,26 +26,43 @@ var AndroidManifest = require('./AndroidManifest');
 var xmlHelpers = require('cordova-common').xmlHelpers;
 var CordovaError = require('cordova-common').CordovaError;
 var ConfigParser = require('cordova-common').ConfigParser;
+var FileUpdater = require('cordova-common').FileUpdater;
 
-module.exports.prepare = function (cordovaProject) {
+module.exports.prepare = function (cordovaProject, options) {
 
     var self = this;
+    var platformResourcesDir = path.relative(cordovaProject.root, path.join(this.locations.root, 'res'));
 
     this._config = updateConfigFilesFrom(cordovaProject.projectConfig,
         this._munger, this.locations);
 
     // Update own www dir with project's www assets and plugins' assets and js-files
-    return Q.when(updateWwwFrom(cordovaProject, this.locations))
+    return Q.when(updateWww.call(self, cordovaProject))
```

Comment:
I'd rather not to pass `this` reference to this method, especially if the only thing we need is a couple of properties from 'locations' object (`this.locations.platformWww` and `this.locations.www` at L120). Maybe just pass these properties as an arguments, or revert to previous signature?